docs/js/index.js

```diff
@@ -59,3 +59,5 @@ require('./with-open-web-component')();
 // TEST MANUALLY (cypress-plugin-tab doesn't support inert)
 // http://localhost:9966/#demo-inert
 require('./inert')();
+
+require('./dom-remove')();
```

Comment:
Just a tiny request to move this up to line 26 now that you've rebased. I reorganized the requires to put all the ones that are Cypress-testable before all those that aren't, just to make it easier for me to follow the "list" when doing manual testing.